File: concrete/src/main/java/pcb/uwu/unit/derived/scalar/SteradianUnit.java
package pcb.uwu.unit.derived.scalar;

public class SteradianUnit extends SolidAngleUnit {

	public static final SteradianUnit STERADIAN = new SteradianUnit();

	public SteradianUnit() {
		super();
	}

	@Override
	public String getSymbol() {
		return "sr";
	}

	@Override
	public String getSingularName() {
		return "steradian";
	}

	@Override
	public String getPluralName() {
		return "steradians";
	}
}

File: implementation/src/main/java/pcb/uwu/unit/derived/geometry/SteradianUnit.kt
package pcb.uwu.unit.derived.geometry

import pcb.uwu.unit.base.RadianUnit

object SteradianUnit
    : SolidAngleUnit(angleUnit1 = RadianUnit,
                     angleUnit2 = RadianUnit)
{
    override val symbol = "sr"
    override val singularName = "steradian"
    override val pluralName = "steradians"
}
